docs/Features/XMLFilePathSupport.md

```diff
@@ -0,0 +1,127 @@
+# XML File Paths Features
+
+XML file path support provides the capability to mark a DOM node (attribute or text) as file path with the `xml.filePathSupport.mappings` settings by using XPath expression :
+
+ * `path/text()` defines the text node of the `path` element.
+ * `item/@path` defines the `path` attribute node of the `item` element.
+
+Once the DOM node is marked as file path, you will benefit with:
+
+ * Files completion.
+
+## Define File path in Text Content with `path/text()`
+
+Given this XML file `items.xml` sample:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<items>
+	<path>path/to/file.xml</path>
+</items>
+```
+
+In this sample:
+
+ * text of `path` tag element `<path>path/to/file.xml</path>` declare a file path. [vscode-xml](https://github.com/redhat-developer/vscode-xml) provides a file path support with the `xml.filePathSupport.mappings` settings. You can declare this settings:
```

Comment:
In this example:

The text within the `path` tag element `<path>path/to/file.xml</path>` represents a file path. The [vscode-xml](https://github.com/redhat-developer/vscode-xml) extension offers file path support through the `xml.filePathSupport.mappings` settings. You can configure this setting as follows:

Reply:
fixed